config/jobs/kubernetes-sigs/release-notes/release-notes-presubmits.yaml

```diff
@@ -0,0 +1,93 @@
+---
+presubmits:
+  kubernetes-sigs/release-notes:
+    - name: build-dev
+      always_run: true
+      decorate: true
+      spec:
+        containers:
+          - image: node:11
+            command:
+              - /bin/bash
+              - -c
+              - >
+                npm install &&
+                npm run $JOB_NAME
```

Comment:
I am absolute noob with npm so sorry for naive question :see_no_evil: but does this require the project to be checked out at `$GOPATH/src/sigs.k8s.io/release-notes` by any chance? By default, the project will be checked out at `$GOPATH/src/github.com/kubernetes/release-notes`.

If yes, this will also need `path_alias: sigs.k8s.io/release-notes`.

I also see that there is some Go code in the repo, but it doesn't use canonical import paths for importing kubernetes projects: https://github.com/kubernetes-sigs/release-notes/blob/e3627c37e54019ae18a889bdce70ff96cdfa7a11/main.go#L28 ...however, the jobs aren't testing Go code here...so I'm not sure how this all fits in, but just wanted to double check. :smile:

Reply:
I think there  should be no path adaption needed right now since we do not use the go part of the project yet. :)